docs/basic/settings/index.rst

```diff
@@ -137,7 +137,7 @@ ADMIN_MODERATE_UPLOADS
     A ``Group Manager`` *can* approve the resource, but he cannot publish it whenever the setting ``RESOURCE_PUBLISHING`` is set to ``True``.
     Otherwise, if ``RESOURCE_PUBLISHING`` is set to ``False``, the resource becomes accessible as soon as it is approved.
 
-AGON_RATINGS_CATEGORY_CHOICES
+PINAX_RATINGS_CATEGORY_CHOICES
```

Comment:
This should be places on the correct alphabetic order, moreover the underline does not cover all the title; this will cause document rendering failure

Reply:
Amended in https://github.com/GeoNode/geonode/pull/5333/commits/0ccf219ae391f878b4fe3f1b170637e1a8fa8a65